config/nrfconnect/chip-module/Kconfig.features

```diff
@@ -21,7 +21,7 @@ if CHIP
 
 config CHIP_QSPI_NOR
 	bool "Enable QSPI NOR feature set"
-	imply NORDIC_QSPI_NOR
+	default y if NORDIC_QSPI_NOR
```

Comment:
I think this symbol becomes pointless now. It was created to enable required nrf/zephyr symbols from Matter. If you want to set it in a reversed order, just use `NORDIC_QSPI_NOR` in .cpp code and remove this symbol at all.